src/components/media.js

```diff
@@ -16,14 +16,32 @@
     } = options;
 
     const titleText = useText(title);
-    const imgUrl = useText(imageSource);
     const videoUrl = useText(videoSource);
     const iframeUrl = useText(iframeSource);
+    const [imgUrl, setImgUrl] = useState(useText(imageSource));
```

Comment:
I don't think you're allowed to use a hook this way (https://reactjs.org/docs/hooks-rules.html). Can you move the `useText` outside of the `useState`, assign the value to a variable and pass that variable to `useState`?